dev/tidb-in-kubernetes/maintain/lightning.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Restore data into TiDB in Kubernetes
```

Comment:
Also, please use the headline style of capitalization for the `title` (1st-level title).

```suggestion
title: Restore Data into TiDB in Kubernetes
```